packages/tea-map-cache/webpack.config.js

```diff
@@ -9,7 +9,9 @@ module.exports = {
     path: path.resolve(__dirname, 'dist'),
     filename: 'index.js'
   },
-  externals: ['aws-sdk'],
+  externals: [
+    '@aws-sdk'
+  ],
```

Comment:
Should this have the `@`? Also what's up with the `@`? 😄

Reply:
@npauzenga As discussed in Slack this was originally changed because we want (in theory) to exclude @aws-sdk (v3) from our packages since it's provided in the Lambda environment for us.

However your question made me think about this more and it's a good thing it did. We're still targeting Node 16 which does _not_ include the v3 SDK in the Lambda runtime. So we can't exclude it.

Also, I'm not convinced that was excluding it the way we thought it was because most of our packages depend on `aws-client` which does not have a webpack configuration. I've added a comment to our Node upgrade ticket (https://bugs.earthdata.nasa.gov/browse/CUMULUS-3433) to make sure this gets re-added and in a way that actually does what we expect.

Reply:
Thanks for clarifying this. I think it makes sense to just remove the V2 external for now.